src/execute.c

```diff
@@ -107,54 +107,105 @@ static struct cmd_func group_commands[] = {
     { NULL,         NULL            },
 };
 
+
+#ifdef PYTHON
+#define SPECIAL_COMMANDS 6
+#else
+#define SPECIAL_COMMANDS 5
+#endif /* PYTHON */
+
+/* Special commands are commands that only take one argument even if it contains spaces */
+static const char special_commands[SPECIAL_COMMANDS][MAX_CMDNAME_SIZE] = {
+    "/avatar",
+    "/nick",
+    "/note",
+#ifdef PYTHON
+    "/run",
+#endif /* PYTHON */
+    "/title",
+    "/sendfile",
+};
+
+/* Returns true if input command is in the special_commands array. */
+static bool is_special_command(const char *input)
+{
+    int s = char_find(0, input, ' ');
+    int i;
```

Comment:
We don't do C99 in toxic?

Reply:
We do, but we didn't used to, so most loops declare counters in c89 style. Will eventually have to go through and change them all.